include/chemfiles/Configuration.hpp

```diff
@@ -8,6 +8,12 @@
 #include <unordered_map>
 
 #include "chemfiles/mutex.hpp"
+#include "chemfiles/periodic_table.hpp"
+
+namespace toml {
+    class value;
```

Comment:
I'm wondering if it would be better to use the property class here over `value`?

Reply:
I don't understand what you mean here. This is a forward declaration of `toml::value`, so that it can be used in the class declaration. As far as I know, there is no `toml::property` class.

Reply:
Whoops, never mind.  I meant the `chemfiles::Property` class. I release this doesn't make sense, ignore my comment.